### PR TITLE
[UI]: Including reasons card in above

### LINKED ISSUE #1900: [UI] include reasons "above the fold" in task runs
**Describe the bug**

The status of each run is shown at the top (and without clicking "More..."), but that information is incomplete without the reason indication -- things like "claim-expired" for example.  Users often miss that information because it is hidden behind "More...".

I propose moving all three of those "pills" onto the same line of the run, at the top.

image

### PR DESCRIPTION
**Pull request by vishakhanihore**
Github Bug/Issue: Fixes #1900

### CODE REVIEW

**Comment by vishakhanihore** [comment]
@djmitche @helfi92 Please Review

**Comment by vishakhanihore** [comment]
> The location of Reason Resolved looks good to me. However, there is one minor issue:
> 
> Navigating to https://deploy-preview-1932--taskcluster-web.netlify.com/tasks/f2WzQXE5RUSVDjPrHZubCQ/runs/0 currently shows:
> 
> Screen Shot 2019-11-09 at 12 48 10 PM
> 
> From [#1900 (comment)](https://github.com/taskcluster/taskcluster/issues/1900#issuecomment-550047618), Reason Resolved and Reason Created should only have a grey label. Right now "FAILED" shows a red.

TBH I forgot about the colour change, I will do this too

**Comment by vishakhanihore** [comment]
@helfi92 I tried to override the default background using 
```js
    reasonResolved: {
      backgroundColor: grey[800],
    },
```
and writing 
```js
<StatusLabel
  state={run.reasonResolved}
  className={classes.reasonResolved}
/>
```
But still the background is not overriding to grey. What am I doing wrong ?

**Comment by helfi92** [comment]
> But still the background is not overriding to grey. What am I doing wrong ?

You might need to do

```jsx
backgroundColor: `${grey[800]} !important`,
```

**Comment by vishakhanihore** [comment]
@helfi92 Please cehck

File: ui/src/components/TaskRunsCard/index.jsx [hunk 1]
```diff
@@ -341,37 +345,21 @@ export default class TaskRunsCard extends Component {
...
-                    </ListItem>
-                  </CopyToClipboard>
+                  <ListItem>
+                    <ListItemText
+                      primary="Reason Resolved"
+                      secondary={
+                        run.reasonResolved ? (
+                          <StatusLabel
+                            state={run.reasonResolved}
+                            className={classes.reasonResolved}
```
Comment (helfi92) [inline ui/src/components/TaskRunsCard/index.jsx]: The grey coloring should also be done for "Reason Created".

Comment (vishakhanihore) [reply ui/src/components/TaskRunsCard/index.jsx]: @djmitche Done

**Comment by vishakhanihore** [comment]
@djmitche Please review, if it looks good I will squash my commits .

**Comment by vishakhanihore** [comment]
@helfi92 @djmitche Please check

**Comment by vishakhanihore** [comment]
> Navigating to http://localhost:5080/tasks/YIPL90OjRISLnqaOmpC2rg shows the labels as yellow:
> 
> Screen Shot 2019-11-14 at 4 19 37 PM

@helfi92 I am not able to access the `Tasks` on my local . Any suggestion , I was earlier able to access and see the tasks run card too !

**Comment by helfi92** [comment]
> @helfi92 I am not able to access the Tasks on my local . Any suggestion , I was earlier able to access and see the tasks run card too !

Last weekend we did a migration that caused https://taskcluster.net to go away. You can use https://firefox-ci-tc.services.mozilla.com as your new taskcluster root URL.

https://github.com/taskcluster/taskcluster/tree/master/services/web-server#running-taskcluster-web-server-locally

**Comment by vishakhanihore** [comment]
@helfi92 Done, Please review and merge 😄

**Comment by vishakhanihore** [comment]
@helfi92 **Screenshot for your reference :**
Screenshot 2019-11-15 at 11 56 35 PM

**Comment by vishakhanihore** [comment]
@helfi92 I dont know why the `queue` test is failing, Although I didnt make any changes to those files 😕

**Comment by vishakhanihore** [comment]
@helfi92 Please review

**Comment by vishakhanihore** [comment]
@helfi92

**Comment by vishakhanihore** [comment]
@helfi92 Please have a look , its complete

**Comment by helfi92** [comment]
I plan on reviewing this when I get the chance.

**Pull request merged**